flyway/src/main/java/io/micronaut/flyway/ValidatePatternTypeConverter.java

```diff
@@ -28,8 +27,11 @@
  * @author Nenad Vico
  * @see org.flywaydb.core.api.pattern.ValidatePattern
  * @since 5.4.1
+ * @deprecated This converter is no longer necessary in conjunction with enhancements to
+ * {@link FlywayConfigurationProperties} that ensure that {@link org.flywaydb.core.api.configuration.FluentConfiguration#ignoreMigrationPatterns(String...)}
+ * will always be used to set ignore-migration-patterns, letting Flyway perform its own type conversion.
  */
-@Singleton
```

Comment:
Runs in too late...

Removing `@Singleton` results in japicmp errors (as the bean definition isn't written)

Reply:
Argh, ok. I'll open a new PR to put it back since it's harmless.

Reply:
https://github.com/micronaut-projects/micronaut-flyway/pull/538